src/types.ts

```diff
@@ -218,8 +219,20 @@ export type WorkflowServeOptions<
    * 3 by default
    */
   retries?: number;
+  /**
+   * Whether the framework should use `content-type: application/json`
+   * in `triggerFirstInvocation`.
```

Comment:
I find this very confusing because we cannot assume anyone is familiar with our internal workings, they will not know what `triggerFirstInvocation`is. To me a much more friendly approach would be directly specifying headers that are then forwarded, those can include the content type but are not limited

Reply:
useJSONContent is not part of the options we allow in `serve` methods for frameworks (see PublicServeOptions). So it won't be visible to users unless they they import `serve` from the route like `import { serve } from "@upstash/workflow"`

The reason we need this parameter is because we used to have publishJSON in `client.trigger`. this didn't work because publishJSON adds `content-type: application/json`, which breaks some frameworks. So we used `publish`. But some frameworks still need the content type to be `application/json`. sveltekit is one of them.

If you have a better alternative to the current implementation, I am all ears.

Reply:
removed useJSONContent from the public API